src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationUnsuccessfulPostGateway.cshtml

```diff
@@ -207,7 +217,9 @@
             </div>
         </div>
     }
-    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+
+
+<hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
```

Comment:
Nothing major, looks like extra line introduced and indentation out of alignment